storefront/modules/auth/components/LoginForm.tsx

```diff
@@ -1,7 +1,6 @@
-import { useForm } from 'react-hook-form';
-import { useEffect } from 'react';
 import Link from 'next/link';
-
+import { useEffect } from 'react';
```

Comment:
We order the imports as follows:

- Absolute imports (from package.json)
- Blank line
- Relative imports

Each section is ordered alphabetically.

Reply:
ok, I'm on it